mozdownload/scraper.py

```diff
@@ -467,13 +468,13 @@ def get_build_info_for_date(self, date, build_index=None):
             # ensure to select the correct subfolder for localized builds
             'L10N': '(-l10n)?' if self.locale_build else '',
             'PLATFORM': '' if self.application not in (
-                        'fennec') else '-' + self.platform
+                'fennec') else '-' + self.platform
         }
 
         parser.entries = parser.filter(regex)
         parser.entries = parser.filter(self.is_build_dir)
 
-        if has_time:
+        if date.strftime('%H-%M-%S') != '00-00-00':
```

Comment:
Why that change? Mind explaining it to me?

Reply:
Sure.
I was going to ask for some help here. I don't know if this is the correct fix.
But multiple `daily_scraper` tests failed with Python 3 with these errors:
https://travis-ci.org/mozilla/mozdownload/jobs/503557491
Something to do with parsing the build time.

Reply:
First we will keep `has_time`, which means that if we need any update in how we check for a valid time, it has to be done where `has_time` gets its value. It would be good if you have a short snippet and run it once with py27 and py36 to see what the difference is.

Reply:
I updated the `has_time` method